hydrocron/api/controllers/timeseries.py

```diff
@@ -117,25 +118,31 @@ def format_json(results: Generator, feature_id, start_time, end_time, exact, dat
                         feature['geometry']['coordinates'].append([float(x), float(y)])
                     if feature_type == 'Point':
                         feature['geometry']['coordinates'] = [float(x), float(y)]
-                i += 1
-                feature['properties']['time'] = datetime.fromtimestamp(
-                    float(t[constants.FIELDNAME_TIME]) + 946710000).strftime(
-                        "%Y-%m-%d %H:%M:%S")
-                feature['properties']['reach_id'] = float(t[constants.FIELDNAME_REACH_ID])
-                feature['properties']['wse'] = float(t[constants.FIELDNAME_WSE])
-                feature['properties']['slope'] = float(t[constants.FIELDNAME_SLOPE])
-                data['features'].append(feature)
+            columns = []
+            if feature_lower == 'reach':
+                columns = constants.REACH_DATA_COLUMNS
```

Comment:
this isn't going to work, these are the lists of columns in the source data to scramble, not the exhaustive list of the columns in the source files. Eg, specifically excluded are things like feature ID, all three different time fields, all p_ fields copied from SWORD, and all of the fields from the filename that we add manually to the database. We can add in separate constants to use for the columns to return

Reply:
Add this one to use for reaches:
```
REACH_ALL_COLUMNS = [
    'reach_id','time','time_tai','time_str','p_lat','p_lon','river_name',
    'wse', 'wse_u', 'wse_r_u', 'wse_c', 'wse_c_u',
    'slope', 'slope_u', 'slope_r_u', 'slope2', 'slope2_u', 'slope2_r_u',
    'width', 'width_u', 'width_c', 'width_c_u',
    'area_total', 'area_tot_u', 'area_detct', 'area_det_u', 'area_wse',
    'd_x_area', 'd_x_area_u',
    'layovr_val', 'node_dist', 'loc_offset', 'xtrk_dist',
    'dschg_c', 'dschg_c_u', 'dschg_csf', 'dschg_c_q',
    'dschg_gc', 'dschg_gc_u', 'dschg_gcsf', 'dschg_gc_q',
    'dschg_m', 'dschg_m_u', 'dschg_msf', 'dschg_m_q',
    'dschg_gm', 'dschg_gm_u', 'dschg_gmsf', 'dschg_gm_q',
    'dschg_b', 'dschg_b_u', 'dschg_bsf', 'dschg_b_q',
    'dschg_gb', 'dschg_gb_u', 'dschg_gbsf', 'dschg_gb_q',
    'dschg_h', 'dschg_h_u', 'dschg_hsf', 'dschg_h_q',
    'dschg_gh', 'dschg_gh_u', 'dschg_ghsf', 'dschg_gh_q',
    'dschg_o', 'dschg_o_u', 'dschg_osf', 'dschg_o_q',
    'dschg_go', 'dschg_go_u', 'dschg_gosf', 'dschg_go_q',
    'dschg_s', 'dschg_s_u', 'dschg_ssf', 'dschg_s_q',
    'dschg_gs', 'dschg_gs_u', 'dschg_gssf', 'dschg_gs_q',
    'dschg_i', 'dschg_i_u', 'dschg_isf', 'dschg_i_q',
    'dschg_gi', 'dschg_gi_u', 'dschg_gisf', 'dschg_gi_q',
    'dschg_q_b', 'dschg_gq_b',
    'reach_q', 'reach_q_b',
    'dark_frac', 'ice_clim_f', 'ice_dyn_f', 'partial_f', 'n_good_nod',
    'obs_frac_n', 'xovr_cal_q', 'geoid_hght', 'geoid_slop',
    'solid_tide', 'load_tidef', 'load_tideg', 'pole_tide',
    'dry_trop_c', 'wet_trop_c', 'iono_c', 'xovr_cal_c',
    'n_reach_up', 'n_reach_dn', 'rch_id_up', 'rch_id_dn', 
    'p_wse', 'p_wse_var', 'p_width', 'p_wid_var', 'p_n_nodes', 'p_dist_out',
    'p_length', 'p_maf', 'p_dam_id', 'p_n_ch_max', 'p_n_ch_mod', 'p_low_slp',
    'cycle_id', 'pass_id', 'continent_id', 'range_start_time', 'range_end_time', 'crid'
] 
```

Reply:
and this for nodes: 

```
NODE_ALL_COLUMNS = [
    'reach_id', 'node_id', 'time', 'time_tai', 'time_str', 
    'lat', 'lon', 'lat_u', 'lon_u', 'river_name',
    'wse', 'wse_u', 'wse_r_u',
    'width', 'width_u',
    'area_total', 'area_tot_u', 'area_detct', 'area_det_u', 'area_wse',
    'layovr_val', 'node_dist', 'xtrk_dist',
    'flow_angle', 'node_q', 'node_q_b',
    'dark_frac', 'ice_clim_f', 'ice_dyn_f', 'partial_f', 'n_good_pix',
    'xovr_cal_q', 'rdr_sig0', 'rdr_sig0_u', 'rdr_pol',
    'geoid_hght', 'solid_tide', 'load_tidef', 'load_tideg', 'pole_tide',
    'dry_trop_c', 'wet_trop_c', 'iono_c', 'xovr_cal_c',
    'p_wse', 'p_wse_var', 'p_width', 'p_wid_var', 'p_dist_out', 'p_length', 
    'p_dam_id', 'p_n_ch_max', 'p_n_ch_mod',
    'cycle_id', 'pass_id', 'continent_id', 'range_start_time', 'range_end_time', 'crid'
]
```